tasks/main.yml

```diff
@@ -75,3 +75,9 @@
      pattern="unbound"
   tags: ["service","unbound"]
 
+- name: verify proper root forwarding
+  when: "{{unbound_configuration.root-hints is defined }}"
+  command: /usr/sbin/unbound-control list_forwards | /usr/bin/awk '/^. IN/ {print 1}'
+  register: listforwards
+  assert: { that: "listforwards != '1'" }
```

Comment:
Will this work if there is more than one '.' forward?  For example, if there's one for 8.8.8.8 and one for 8.8.4.4 that should print out:
```
1
1
```

A '!=' would cause the assert for that to succeed would it not?